Jenkinsfile

```
@@ -1,6 +1,5 @@
 buildPlugin(configurations: [
         [ platform: "linux", jdk: "8", jenkins: null ],
-        [ platform: "linux", jdk: "11", jenkins: "2.249" ],
-        [ platform: "windows", jdk: "8", jenkins: "2.235.3", javaLevel: "8" ],
-        [ platform: "linux", jdk: "11", jenkins: "2.235.3", javaLevel: "8" ]
-    ])
+        [ platform: "linux", jdk: "11", jenkins: null ],
+        [ platform: "windows", jdk: "8", jenkins: null, javaLevel: "8" ],
```

`jenkins` defaults to `null`, and `javaLevel` should be considered deprecated (no reason to ever use it with any plugin using a remotely recent POM).

```suggestion
        [ platform: "linux", jdk: "11" ],
        [ platform: "windows", jdk: "8" ],
```